### ISSUE TITLE
Incorrect example for onStubActivation on Usage Examples page

### ISSUE THREAD

**Issue by kirilstrax** [AliSoftware/OHHTTPStubs#333]
### New Issue Checklist

- [x] I have tried with the latest version of OHHTTPStubs
- [x] I have read the [README](https://github.com/AliSoftware/OHHTTPStubs/blob/master/README.md)
- [x] I have read the [Using the right Swift Version of `OHHTTPStubs` for your project](https://github.com/AliSoftware/OHHTTPStubs#using-the-right-swift-version-of-ohhttpstubs-for-your-project) section
- [x] I have searched in the [existing issues](https://github.com/AliSoftware/OHHTTPStubs/issues?utf8=✓&q=is%3Aissue)
- [x] I have read [the OHHTTPStubs wiki](https://github.com/AliSoftware/OHHTTPStubs/wiki) to see if there wasn't a detailed page talking about my issue

### Environment

- version of OHHTTPStubs: 8.0.0
- integration method you are using:
  * [x] Cocoapods
  * [ ] Carthage
  * [ ] submodule
  * [ ] other
- version of the tool you use: not applicable

### Issue Description

The example on Usage Examples page is shown as follows:
```
// Swift
OHHTTPStubs.onStubActivation() { request, stub in
  println("\(request.URL!) stubbed by \(stub.name).")
}
```

*Problem 1*: Signature of `onStubActivation` was changed a few versions ago. So it should be:
```// Swift
OHHTTPStubs.onStubActivation() { request, stub, response in
```
This problem applies to both, Swift and Objective C examples

*Problem 2*: replace `println` with `print` in Swift, as there's no println function in swift.

*Problem 3*: in Swift, `NSURLRequest` is bridged as `URLRequest`, and instead of `request.URL!`, it uses lower-cased `request.url!`

Bottom line: example in Swift should look something like this:

```
// Swift
OHHTTPStubs.onStubActivation() { request, stub, response in
  print("\(request.url!) stubbed by \(stub.name) of size \(response.dataSize).")
}
```

##### Complete output when you encounter the issue (if any)

This is documentation update issue, so not applicable.
